lxc/storage_volume.go

```diff
@@ -1379,6 +1400,7 @@ func (c *cmdStorageVolumeMove) Command() *cobra.Command {
 
 	cmd.Flags().StringVar(&c.storageVolumeCopy.flagMode, "mode", "pull", i18n.G("Transfer mode, one of pull (default), push or relay")+"``")
 	cmd.Flags().StringVar(&c.storage.flagTarget, "target", "", i18n.G("Cluster member name")+"``")
+	cmd.Flags().StringVar(&c.storageVolume.flagDestinationTarget, "destination-target", "", i18n.G("Destination cluster member name")+"``")
```

Comment:
This change introduces a new translatable string in the client, so you'll need to do:
 - make i18n
 - git commit -a -s -m "i18n: Update translation templates"

That will get you an additional commit with just the translation changes.

Reply:
Ah neat! Fixed below.